mautic/mautic
# Allow console to leverage the generic autoloader regardless of the location

Allow console to leverage the generic autoloader regardless of the location

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
 -->

When using the recommended-project approach, the bin/console is not using the correct path. We now have a generic autoloader component, so also no need to manually point at the AppKernel file as it is already included.

#### Steps to test this PR:
1. Install using mautic/recommended-project approach
2. Try bin/console in the CLI
3. See it fail
4. Apply this patch
5. It should work now
